package.json

```diff
@@ -17,8 +17,19 @@
     "eslint": "^9.6.0",
     "globals": "^15.8.0",
     "husky": "^9.0.11",
+    "lint-staged": "^15.2.7",
     "parcel": "2.12",
     "prettier": "^3.3.2",
     "sass": "^1.77.6"
+  },
+  "lint-staged": {
```

Comment:
what about css? no need to use prettier on CSS files?

Also checkout https://stylelint.io/ for linting css

Reply:
Fixed, added .sass extension instead, as css file will be auto-generated and compressed